apps/ui/src/components/ProposalVoteApproval.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { Choice, Proposal } from '@/types';
+
+defineProps<{
+  sendingType: Choice | null;
+  proposal: Proposal;
+}>();
+
+const emit = defineEmits<{
+  (e: 'handleVoteClick', value: Choice);
```

Comment:
```suggestion
  (e: 'voteClick', value: Choice);
```

We tend to call handler function with `handle` prefix, but emit names don't have that prefix.

Reply:
Do we need to keep the `Click` suffix ? If not, it will look better like

```
@vote="handleVoteClick"
```

Reply:
Also, we have a `handleVoteClick` function in 2 places: ProposalListsItem.vue and Proposal.vue. 

Wouldn't it be cleaner if we move this inside ProposalVote ?